pages/_lang/publish.vue

```diff
@@ -805,12 +805,217 @@
+                      required
+                      v-model="windowsForm.version"
+                    />
+                  </div>
+                </div>
+
+              </div>
+
+              <div class="row">
+                <div class="col-lg-6 col-md-12">
+                  <div class="form-group">
+                    <label for="windowsClassicAppVersionInput">
+                      Classic Package version
+                      <i
+                        class="fas fa-info-circle"
+                        title="The version of your app displayed to users on older version of Windows 10"
+                        aria-label
+                        role="definition"
+                      ></i>
+                    </label>
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="windowsClassicAppVersionInput"
+                      placeholder="1.0.0"
+                      required
+                      v-model="windowsForm.classicPackage.version"
+                    />
+                  </div>
+                </div>
+
+              </div>
+
+              <div class="form-group">
+                <label for="windowsStartUrlInput">
+                  URL
+                  <i
+                    class="fas fa-info-circle"
+                    title="This is the URL for your PWA."
+                    aria-label="This is the URL for your PWA."
+                    role="definition"
+                  ></i>
+                </label>
+                <input
+                  type="url"
+                  class="form-control"
+                  id="windowsStartUrlInput"
+                  placeholder="/index.html"
+                  required
+                  v-model="windowsForm.url"
+                />
+              </div>
+
+              <div class="form-group">
+                <label for="windowsManifestUrlInput">Manifest URL</label>
+                <input
+                  type="url"
+                  class="form-control"
+                  id="windowsManifestUrlInput"
+                  placeholder="https://mysite.com/manifest.json"
+                  required
+                  v-model="windowsForm.manifestUrl"
+                />
+              </div>
+
+            </div>
+
+            <!-- right half of the options dialog -->
+            <div class="col-lg-6 col-md-12">
+              <div class="form-group">
+                <label for="windowsIconUrlInput">Icon URL</label>
+                <input
+                  type="url"
+                  class="form-control"
+                  id="windowsIconUrlInput"
+                  placeholder="https://myawesomepwa.com/512x512.png"
+                  v-model="windowsForm.images.baseImage"
+                />
+              </div>
+
+              <div class="form-group">
+                  <label for="windowsDisplayNameInput">
+                    Display Name
```

Comment:
Can you change this to `Publisher Display Name`? And make it a link to https://github.com/pwa-builder/pwabuilder-windows-chromium-docs/blob/master/find-publisher.md